op-preimage/oracle.go

```diff
@@ -4,6 +4,8 @@ import (
 	"encoding/binary"
 	"fmt"
 	"io"
+
+	"github.com/ethereum/go-ethereum/common"
```

Comment:
`op-preimage` is used by Cannon example programs which otherwise don't depend on geth. Can we avoid the geth import, and just hex-encode using standard utils instead?

Reply:
ah, that would explain the cannon failure.  Nice to see that doing its job. Will fix.